packages/adapter-nextjs/__tests__/auth/createAuthRouteHandlersFactory.test.ts

```diff
@@ -75,45 +75,49 @@ describe('createAuthRoutesHandlersFactory', () => {
 		mockIsValidOrigin.mockReturnValue(true);
 	});
 
-	it('throws an error if the `amplifyAppOrigin` param has value of `undefined`', () => {
-		expect(() =>
-			createAuthRouteHandlersFactory({
+	describe('the created createAuthRouteHandlers function', () => {
+		it('throws an error if the AMPLIFY_APP_ORIGIN environment variable is not defined', () => {
+			const throwingFunc = createAuthRouteHandlersFactory({
```

Comment:
Nit: Reflect imagined DX/best practice
```suggestion
			const createAuthRouteHandlers = createAuthRouteHandlersFactory({
```